### PR TITLE
README.md - Link to demo page and changes to table

### PR DESCRIPTION
**Pull request by justice47**
- Added link to demo page
- Changed formatting in table with props

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,7 +67,7 @@ and then include it in your Sass/CSS files
 Include the component,
 
 ```javascript
-import DataTable from 'vue-materialize-datatable';
+import DataTable from "vue-materialize-datatable";
```
Comment (MicroDroid) [inline README.md]: Hmm, no, make this back single quotes.

File: README.md [hunk 2]
```diff
@@ -85,47 +90,256 @@ And then.. use the component:
...
+    </td>
+    <td>
+      Rows
+    </td>
+    <td>
+      <pre>
+        <code>
+[ // Array of objects 
+  {    
+    name: "test", // Whatever
```
Comment (MicroDroid) [inline README.md]: Single quotes again

**Comment by MicroDroid** [comment]
You must merge the GitHub pages demo branch to here, aside from this.

**Comment by justice47** [comment]
> You must merge the GitHub pages demo branch to here, aside from this.

Hi! How can I make this? I can only make a PR, demo page made as a standalone vue app, so should I make PR to some branch inside main project (vue-materialize-datatable)?

**Comment by MicroDroid** [comment]
@justice47 Hi, sorry for the delayed response, I read the email and forgot to reply.

Yes, you make a branch called `gh-pages`, push stuff there and make a PR for it. Make sure to include source and not just compiled files, as well.

**Comment by justice47** [comment]
@MicroDroid no problem!
I've created new branch in my fork of **vue-materialize-datatable**, but then I'm trying to make a PR to your master branch, github shows message:
"There isn’t anything to compare.
MicroDroid:master and justice47:gh-pages are entirely different commit histories."
I think you should create empty branch, and then I can push gh-pages branch to it.

**Comment by MicroDroid** [comment]
Alright created the branch, try now see if it works.

**Comment by MicroDroid** [comment]
Wait this wouldn't work, uhh, will have a further look in a little while

**Comment by MicroDroid** [comment]
@justice47 Hey there, you can put everything in `docs/` folder with the `master` branch be done with it, no need for extra branches. Just push changes to this PR maybe.

Now for the source, maybe put it in a directory outside `docs/`, no idea where just find something appropriate, and put compiled stuff in `docs/` so I configure GitHub to serve from it.

Thanks for everything and sorry for BIG delays.

**Comment by justice47** [comment]
@MicroDroid hi!
I've added docs dist and docs src in new commit.

**Comment by MicroDroid** [comment]
heh I'm too lazy to review this PR.

I'll blindly merge it; I hope it works well enough :eyes:

**Comment by MicroDroid** [comment]
Alright there were a few issues, got rid of them and fixed them up. It's up now on https://microdroid.github.io/vue-materialize-datatable/

Thanks a lot @justice47!

**Pull request merged**